src/process.h

```diff
@@ -201,6 +201,8 @@ struct Lisp_Process
     bool_bf gnutls_complete_negotiation_p : 1;
 #endif
 };
+/* Accessors for Rust */
+pid_t fget_pid(const struct Lisp_Process *p);
```

Comment:
Follow the C style. The return type is on a line then the rest of the definition is on the next.

Reply:
I did not noticed that style from PR  #427. But sure I'll follow the C style.